src/pandas_profiling/model/describe.py

```diff
@@ -571,87 +63,80 @@ def describe(df: pd.DataFrame) -> dict:
 
     disable_progress_bar = not config["progress_bar"].get(bool)
 
-    # Multiprocessing of Describe 1D for each column
-    pool_size = config["pool_size"].get(int)
-    if pool_size <= 0:
-        pool_size = multiprocessing.cpu_count()
-
-    args = [(column, series) for column, series in df.iteritems()]
-    series_description = {}
-    with tqdm(total=len(args), desc="variables", disable=disable_progress_bar) as pbar:
-        if pool_size == 1:
-            for arg in args:
-                column, description = multiprocess_1d(arg)
-                series_description[column] = description
-                pbar.update()
-        else:
-            # Store the original order
-            original_order = {
-                k: v for v, k in enumerate([column for column, _ in args])
-            }
+    correlation_names = [
+        correlation_name
+        for correlation_name in [
+            "pearson",
+            "spearman",
+            "kendall",
+            "phi_k",
+            "cramers",
+            "recoded",
+        ]
+        if config["correlations"][correlation_name]["calculate"].get(bool)
+    ]
+
+    number_of_task = 7 + len(df.columns) + len(correlation_names)
+
+    with tqdm(
+        total=number_of_task, desc="Describe", disable=disable_progress_bar
```

Comment:
"Describe" might not be the most informative description. How about "Profiling"?

Reply:
I've noticed many use of the statement 'Describe', like ''description_set", "describe_df", so I think maybe 'describe' can better continue the previous naming style.